.github/CONTRIBUTING.md

```diff
@@ -41,7 +24,7 @@ Here are a few guidelines to follow when submitting a pull request:
 1. Create a branch that lightly defines what you're working on (e.g. add-styles).
 1. Ensure that your contribution works via `npm`, if applicable. See below under
    _Install the package locally via `npm-link`_.
-1. Once you're ready to submit a pull request, push your branch up to the repo.
+1. Once you're ready to submit a pull request and fill out the PULL REQUEST template provided.
```

Comment:
Makes sense! I just wanted it to link to something rather than just being text. Also didn't see your message before I posted.